src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/high-risk-data-breaches/highRiskBreachData.tsx

```diff
@@ -117,9 +142,21 @@ function getDoneStepContent(
               "fix-flow-celebration-high-risk-description-next-security-recommendations",
             )}
           </p>
-          <Button variant="primary" small href={nextStep.href} autoFocus={true}>
+          <TelemetryButton
+            event={{
+              module: "ctaButton",
+              name: "click",
+              data: {
+                button_id: "continue_celebrate_high_risk",
+              },
+            }}
+            variant="primary"
+            small
+            href={nextStep.href}
+            autoFocus={true}
```

Comment:
Question: Will this interfere with the focus changes for the step indicator in #4080?